dockerfiles/frameworks/src/Dockerfile

```diff
@@ -26,36 +26,50 @@ RUN set -xe; \
         php-xdebug \
         php-zip \
         php-xmlwriter \
-        php-redis \
         php-mysql \
         php-sqlite3 \
         php-memcached \
         php-amqp \
+        php-dev \
+        php-pear \
         curl \
-        vim;
+        vim; \
+    pecl install redis
 
 RUN set -xe; \
     php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');";\
     php -r "if (hash_file('sha384', 'composer-setup.php') === 'a5c698ffe4b8e849a443b120cd5ba38043260d5c4023dbf93e1558871f1f07f58274fc6f4c93bcfd858c6bd0775cd8d1') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"; \
     php composer-setup.php; \
     php -r "unlink('composer-setup.php');"; \
     mv composer.phar /usr/local/bin/composer;
+
 ENV COMPOSER_MEMORY_LIMIT -1
 ADD entrypoint.sh /usr/local/bin/entrypoint.sh
+ADD 20-redis.ini /etc/php/7.3/cli/conf.d/20-redis.ini
+
 ENTRYPOINT [ "/usr/local/bin/entrypoint.sh" ]
 CMD [ "bash" ]
 
 FROM base AS symfony
+ENV COMPOSER_MEMORY_LIMIT -1
+ARG SYMFONY_VERSION_TAG=v4.3.4
 
-ARG SYMFONY_VERSION_TAG=v4.3.3
 RUN set -xe; \
     git clone --depth 1 --branch ${SYMFONY_VERSION_TAG} https://github.com/symfony/symfony /home/symfony; \
     cd /home/symfony; \
-    export COMPOSER_MEMORY_LIMIT=-1; \
-    composer install --prefer-dist; \
-    php ./phpunit --list-tests
-
+    composer update --prefer-dist; \
+    php ./phpunit install
 WORKDIR /home/symfony
+ADD symfony_4.3.4.patch ./
+
+RUN set -xe; \
+  git add -f vendor; \
+  git config --global user.email "you@example.com"; \
+  git config --global user.name "Your Name"; \
+  git commit -m "vendor"; \
+  git apply symfony_4.3.4.patch
```

Comment:
I would suggest to either use arg `SYMFONY_VERSION_TAG` or give it a more generic name to the file without any version tag.